model/src/main/kotlin/config/ProviderConfiguration.kt

```diff
@@ -23,25 +23,28 @@ import com.sksamuel.hoplite.ConfigAlias
 
 import org.ossreviewtoolkit.utils.common.Plugin
 
-data class PackageCurationProviderConfiguration(
+/**
+ * The configuration of a package curation and package configuration provider.
+ */
+data class ProviderConfiguration(
```

Comment:
Also, is the class name too generic now? Should we maybe mention the plugin context by naming this e.g. `ProviderPluginConfiguration`, and maybe even moving it next to `PluginManager.kt`?

Note that I'm just thinking out aloud; I do not want to imply it's the right thing to do, but just start a discussion.

Reply:
> Also, is the class name too generic now?

Yes, I also thought that it might be too generic.

> Should we maybe mention the plugin context by naming this e.g. ProviderPluginConfiguration, and maybe even moving it next to PluginManager.kt?

I like `ProviderPluginConfiguration`. But I think we should keep it in the `model` module, as this is clearly a model, and moving it to `utils` would also break existing curation provider plugins.

Reply:
Renamed the class.